### PR TITLE
Added Developer Guide

### PR DESCRIPTION
**Pull request by paoloantinori**

### CODE REVIEW

File: DevelopersGuide.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+sudo dnf config-manager --add-repo https://dl.yarnpkg.com/rpm/yarn.repo
+sudo dnf install -y yarn
+```
+1. build `syndesis-ui`
+```
+cd syndesis-ui
+yarn install
+cd scripts
+sh minishift-setup.sh
+sh syndesis-install
```
Comment (gashcrumb) [inline DevelopersGuide.md]: Switch these two commands around, i.e. `syndesis-install` and then `minishift-setup.sh`

Technically we could even consider adding yarn targets for both scripts, like `yarn setup:minishift`

File: DevelopersGuide.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+cd atlasmap-ui
+yarn install ; yarn start
+# it will open a browser tab, connected to the standalone runtime you can interact with
+# it will save the corresponding xml mapping file in `atlasmap/atlasmap-runtime/runtime/target/mappings/` where you can inspect the produced file 
+```
+
+###### In full integration with Syndesis
+- ends up using the steps for Web Development 
+```bash
+# after you have built atalsmap-ui project with yarn
```
Comment (gashcrumb) [inline DevelopersGuide.md]: Actually no need to build atlasmap-ui at all, this occurs when you build the main syndesis project.

**Review by gashcrumb — CHANGES_REQUESTED** [review]
Just a couple little things, other than that looks good!

**Comment by paoloantinori** [comment]
Done. Would benefit from a squash merge.

**Comment by kahboom** [comment]
Cool! Thanks @paoloantinori

**Comment by rhuss** [comment]
Thanks @paoloantinori for writing this done, very helpful.

I hope we can setup a comprehensive document (asciidoc) in syndesisio/syndesis which combines various aspects into a single document.

**Review by rhuss — APPROVED** [review]
🏑

**Pull request merged**